wallet/src/signal.rs

```diff
@@ -5,14 +5,19 @@ use actix::prelude::*;
 pub fn ctrl_c<T: Fn() + 'static>(cb: T) {
     #[cfg(unix)]
     let sigterm = tokio_signal::unix::Signal::new(tokio_signal::unix::SIGTERM)
-        .map(|s| s.map(|_| ()))
+        .map(|s| {
+            log::trace!("Received SIGTERM signal");
```

Comment:
This is received when doing `kill $(pidof witnet)`, removing this handler will make the wallet shutdown instantly and corrupt the database.